### ISSUE TITLE
Raise the gasLimit to match other networks

### ISSUE THREAD

**Issue by chatch** [ethereum/ropsten#19]
The ropsten gasLimit remains at around 4.7million.

However other chains have much higher limits now:
  mainnet: ~8
  kovan: ~7
  rinkeby: ~7.3

Can we have the miners raise the limit?

Most recent blocks are mined by these 2 accounts:
  0x00d8ae40d9a06d0e7a2877b62e32eb959afbe16d
  0x122932e3159bd5190e0dbcef7ba59a6c041ae9da

Who are these miners?

**Comment by gmaijoe** [comment]
Agreed! i have issues deploying testnet contracts because the gas limit is exceeded in Ropsten...

**Comment by ejossev** [comment]
Agree. The current limit is way too low to do any serious testing. Isn't it the point of the testnet?

**Comment by branciard** [comment]
Event at 4700000, i can't deploy : exceeds block gas limit . Where can we see the current gasLimit in ropsten ? I suppose also it is due to ropsten spammers that take part of the gas per block... annoying.

**Comment by AC0DEM0NK3Y** [comment]
Adding a +1. For close to max size contracts it is miner dependent as to whether your deployment passes which hurts testing time.

**Comment by kmiyachi** [comment]
+1

**Comment by ramxis** [comment]
+1
I have been having same issues, ropsten gas limit should mimick that of the main network.

**Comment by anymos** [comment]
Not sure why it is taking so long ...
